apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/StorageClassTest.cs

```diff
@@ -103,7 +104,7 @@ public void CreateBucketAndObjects()
             string objectName2 = IdGenerator.FromGuid();
 
             var obj2 = client.UploadObject(bucketName, objectName2, "application/octet-stream", TestHelpers.GenerateData(100));
-            Assert.Equal(updatedBucketStorageClass, obj2.StorageClass);
+            Assert.NotEqual(updatedBucketStorageClass, obj2.StorageClass);
```

Comment:
Is this change intentional? My guess is that it isn't.

Reply:
No It's not intended. I'm reverting it.

Reply:
Removed now.